distributor/pkg/uniform/controlplane/controlplane.go

```diff
@@ -66,7 +66,7 @@ func (c *ControlPlane) createRegistrationData() models.Integration {
 	if c.env.Location == "" {
 		location = config.ConnectionTypeToLocation[c.connectionType]
 	} else {
-		location = c.env.Location
+		location = c.env.K8sDeploymentComponent
```

Comment:
i believe we can revert this back to `location = c.env.Location` - if `env.Location` is not set explicitly, the location will be set to either `control-plane` or `remote-execution-plane`, so I think using the name of the component, i.e. the service might not be the correct value for this field

Reply:
reverted 👍🏼